grakn-engine/src/main/java/ai/grakn/engine/controller/CommitLogController.java

```diff
@@ -49,9 +47,11 @@
  */
 //TODO Implement delete
 public class CommitLogController {
+    private final String defaultKeyspace;
```

Comment:
I suspect that this can actually be removed. Future PR though as not relevant to the objectives of this PR.